themes/cerveza-dark/components/OutlinedInput.ts

```diff
@@ -12,6 +14,10 @@ const OutlinedInput = {
                     color: color.gray.lightest,
                 },
 
+                '& input$disabled': {
+                    color: color.gray.darker,
```

Comment:
What is this fixing? This looks like it will either be overridden or overwrite my change I put in that you merged.

I think I had something like 

'$disabled': { '& input': { color: color.blueGrey.dark } }